### PR TITLE
Week 2 assignments

### PR DESCRIPTION
**Pull request by konarssuresh**
This PR consisits changes for week 2 assignments
* code changes for exercise 2.1 in which we have to create webpage .(you can use live-server to view this or else you can directly open index.html in browser)
image
image
image

* code changes for animation exercise . [preview](https://htmlpreview.github.io/?https://github.com/pesto-students/p4-divjot-konarssuresh/week_2_assignments/Week-2/animation-exercise/index.html)

Request you to review it when feasible

### CODE REVIEW

File: Week-2/animation-exercise/styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+.parent {
+  width: 20rem;
+  height: 20rem;
+  margin: auto;
+  position: relative;
+  text-align: center;
+  vertical-align: middle;
+}
+
+.child {
```
Comment (divjot-singh) [inline Week-2/animation-exercise/styles/styles.css:24]: please add z-index

File: Week-2/portfolio-exercise/index.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
```
Comment (divjot-singh) [inline Week-2/portfolio-exercise/index.html:1]: 1. Add margin between 2 article columns for desktop.
2. For single column layout, make article text center aligned
3. For single column layout, make footer text center aligned
4. Margin between footer text and the "article.text-center"
5. Margin between labels and inputs.
6. Make input form buttons center aligned for single column.
7. Make radio buttons center aligned for single column.

**Review by divjot-singh — APPROVED** [review]

File: Week-2/portfolio-exercise/index.html (hunk 2 above)
Comment (konarssuresh) [reply Week-2/portfolio-exercise/index.html:1]: fixed

File: Week-2/animation-exercise/styles/styles.css (hunk 1 above)
Comment (konarssuresh) [reply Week-2/animation-exercise/styles/styles.css:24]: fixed

**Pull request merged**